### PR TITLE
Add product security stuff.

### PR DESCRIPTION
**Pull request by ggallen**

### CODE REVIEW

**Comment by ggallen** [comment]
@apahim, could you please review?

I'm not sure about this change in particular:

`gopkg.in/stretchr/testify.v1 v1.2.2 // indirect`

There is already this in the the file:

` github.com/stretchr/testify v1.6.1`

**Comment by ggallen** [comment]
@apahim, some context for this PR....

Product security is becoming part of the CPaaS pipeline, and as such they need to scan code and report vulnerabilities. These changes add some code to the test product that won't actually be built but will be scanned (and hopefully caught) by the product security pipeline.

So these changes actually bring in stuff with vulnerabilities.

File: go.mod [hunk 1]
```diff
@@ -3,12 +3,204 @@ module github.com/openshift/reference-addon
...
+	gopkg.in/asn1-ber.v1 v1.0.0-20150924051756-4e86f4367175 // indirect
+	gopkg.in/editorconfig/editorconfig-core-go.v1 v1.3.0 // indirect
+	gopkg.in/gomail.v2 v2.0.0-20160411212932-81ebce5c23df // indirect
+	gopkg.in/ini.v1 v1.55.0 // indirect
+	gopkg.in/ldap.v3 v3.0.2 // indirect
+	gopkg.in/square/go-jose.v2 v2.3.1 // indirect
+	gopkg.in/src-d/go-billy.v4 v4.3.2 // indirect
+	gopkg.in/src-d/go-git-fixtures.v3 v3.5.0 // indirect
+	gopkg.in/src-d/go-git.v4 v4.11.0 // indirect
+	gopkg.in/stretchr/testify.v1 v1.2.2 // indirect
```
Comment (apahim) [inline go.mod:192]: this will be resolved to v1.6.1, as that's the max minor between the two

Comment (apahim) [reply go.mod:192]: not a problem, I'd say

File: main.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	fmt.Printf("MD4 is the new MD5: %x\n", h.Sum(nil))
+
+	err := ioutil.WriteFile("config/phish-config.json", validConfig, 0644)
+	conf := config.Config{}
+	fmt.Printf("GONE PHISH'N for configs %v, maybe error: %v\n", conf, err)
+
+	fmt.Printf("IS A README?? : %v as per gitea\n", markup.IsReadmeFile("README.md"))
+
+	fmt.Println("HI I'M INTENTIONALLY USING VULNERABLE LIBS")
+}
```
Comment (apahim) [inline main.go:45]: Not sure how you intend to use this, but it should probably go in the https://github.com/continuous-prod/reference-addon/blob/main/cmd/reference-addon-manager/main.go, so you see what's happening when running this as an operator

Comment (ggallen) [reply main.go:45]: @themenucha, can you respond here?  I don't think this is an issue as you don't care about this at run time.

**Comment by themenucha** [comment]
@apahim , please take a look at the new PR : https://github.com/continuous-prod/reference-addon/pull/20, it won't affect running in any case.Thank you

**Pull request closed without merging**